cyclonedx-bom/src/models/metadata.rs

```diff
@@ -38,6 +38,21 @@ pub struct Metadata {
     pub properties: Option<Properties>,
 }
 
+impl Metadata {
+    pub fn new() -> Self {
+        Self {
+            timestamp: DateTime::now().ok(),
```

Comment:
We should probably have an issue to figure out logging and link this to it so we don't silently hide timestamp issues